### ISSUE TITLE
I can't get my read transparent like in my other phone

### ISSUE THREAD

**Issue by demon456blade** [mihonapp/mihon#1157]
### Steps to reproduce

Download the app
Restore back up
Read a chapter 
![Screenshot_20240825_103738](https://github.com/user-attachments/assets/be37f5b7-9b63-4d02-bdb7-6384da51e5ba)

### Expected behavior

What should happen is the chapter look like this after you read it
![Screenshot_2024-08-25-10-38-58-113](https://github.com/user-attachments/assets/d3af9c5b-e5e8-4aab-b04d-9fe60f884ab7)

### Actual behavior

What actually happens is it looks like this
![Screenshot_20240825_103738](https://github.com/user-attachments/assets/6911a3eb-ffd9-4f58-bf01-791d167c4e2d)

### Mihon version

0.16.5

### Android version

Android 14

### Device

Red magic 9s pro

### Other details

The thing is this is just happening on my red magic 9s pro and not on my red magic 7s pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by demon456blade** [comment]
I just tried it and it still does not work ,I tried it with multiple different colors and restarting the app> Try changing the Theme/colors in Mihon Settings > Appearance, and restarting the app, to see if that fixes the color/read issue.

**Comment by Smol-Ame** [comment]
On your other phone, try creating a new backup file without including App Settings. Then, clear the data on this phone (the one with the color issue), restore the newly made backup, and see if that helps

**Comment by demon456blade** [comment]
> On your other phone, try creating a new backup file without including App Settings. Then, clear the data on this phone (the one with the color issue), restore the newly made backup, and see if that helps

I will try it rn

**Comment by demon456blade** [comment]
> On your other phone, try creating a new backup file without including App Settings. Then, clear the data on this phone (the one with the color issue), restore the newly made backup, and see if that helps
 
It seems that it's still not working

**Comment by Smol-Ame** [comment]
Last thing I can think of for you to try is download Mihon Beta (it's a separate app compared to Mihon Stable), restore your backup in there, and see if the colors are fixed.

Mihon Beta has some changes around the colors/Theme, along with dependency updates that might help work with your device better.

**Comment by demon456blade** [comment]
> Last thing I can think of for you to try is download Mihon Beta (it's a separate app compared to Mihon Stable), restore your backup in there, and see if the colors are fixed.
> 
> Mihon Beta has some changes around the colors/Theme, along with dependency updates that might help work with your device better.

It's not working do you thing there might be something wrong with my device itself

**Comment by Smol-Ame** [comment]
Not sure, maybe it's not playing well with Compose or something. I guess make sure everything is up to date, OS, security, app store.

Just checking, are you using a custom ROM or anything?

**Comment by demon456blade** [comment]
> Not sure, maybe it's not playing well with Compose or something. I guess make sure everything is up to date, OS, security, app store.
> 
> Just checking, are you using a custom ROM or anything?

I don't believe so as I am using the ROM that comes with the red magic 9s pro ,but this is a phone that is made in China but the 7s pro which I also bought from them has none of these problems just this 9s pro

**Comment by MajorTanya** [comment]
@demon456blade Does this still occur on the current Mihon version (0.17.1) or current beta (r7079)?

**Comment by demon456blade** [comment]
> @demon456blade Does this still occur on the current Mihon version (0.17.1) or current beta (r7079)?

What I did to fix it was format my phone and reinstall in ,that fixed it

**Comment by MajorTanya** [comment]
> > [@demon456blade](https://github.com/demon456blade) Does this still occur on the current Mihon version (0.17.1) or current beta (r7079)?
> 
> What I did to fix it was format my phone and reinstall in ,that fixed it

I guess that is one way to fix something like this. I'll close this as resolved then.
